acvm/src/pwg/mod.rs

```diff
@@ -350,8 +347,8 @@ pub fn default_is_opcode_supported(language: Language) -> fn(&Opcode) -> bool {
     // The ones which are not supported, the acvm compiler will
     // attempt to transform into supported gates. If these are also not available
     // then a compiler error will be emitted.
-    fn plonk_is_supported(opcode: &Opcode) -> bool {
```

Comment:
This and the codeblock that it is in, is part of a deprecated feature, so we should remove that in a separate PR and update Noir to no longe rely on these deprecated method